runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -80,6 +80,9 @@
     long mNativeContent;
     long mNativeWebContents;
 
+    // TODO(hengzhi.wu): This should be in a global context, not per XWalkView.
+    private double mDIPScale;
```

Comment:
I suggest not. DIP means device independent pixels. Not a English word dip. Also, android webview has such a style.